deepchecks/vision/checks/methodology/simple_feature_contribution.py

```diff
@@ -0,0 +1,235 @@
+            '',
+            '<u>In the graph above</u>, we should suspect we have problems in our data if:',
+            ''
+            '1. <b>Train dataset PPS values are high</b>:',
+            '   A high PPS (close to 1) can mean that there\'s a bias in the dataset, as a single property can predict'
+            '   the label successfully, using simple classic ML algorithms',
+            '2. <b>Large difference between train and test PPS</b> (train PPS is larger):',
+            '   An even more powerful indication of dataset bias, as an image property that was powerful in train',
+            '   but not in test can be explained by bias in train that is not relevant to a new dataset.',
+            '3. <b>Large difference between test and train PPS</b> (test PPS is larger):',
+            '   An anomalous value, could indicate drift in test dataset that caused a coincidental correlation to '
+            'the target label.'
+        ]
+
+        ret_value, display = get_single_feature_contribution(df_train,
+                                                             'target',
+                                                             df_test,
+                                                             'target',
+                                                             self.ppscore_params,
+                                                             self.n_top_properties)
+
+        if display:
+            display += text
+
+        return CheckResult(value=ret_value, display=display, header='Simple Feature Contribution')
+
+    def add_condition_feature_pps_difference_not_greater_than(self: SFC, threshold: float = 0.2) -> SFC:
+        """Add new condition.
+
+        Add condition that will check that difference between train
+        dataset property pps and test dataset property pps is not greater than X.
+
+        Parameters
+        ----------
+        threshold : float , default: 0.2
+            train test ps difference upper bound.
+
+        Returns
+        -------
+        SFC
+        """
+
+        def condition(value: Dict[Hashable, Dict[Hashable, float]]) -> ConditionResult:
+            failed_features = {
+                feature_name: format_number(pps_diff)
+                for feature_name, pps_diff in value['train-test difference'].items()
+                if pps_diff > threshold
+            }
+
+            if failed_features:
+                message = f'Features with PPS difference above threshold: {failed_features}'
+                return ConditionResult(False, message)
+            else:
+                return ConditionResult(True)
+
+        return self.add_condition(f'Train-Test properties\' Predictive Power Score difference is not greater than '
+                                  f'{format_number(threshold)}', condition)
+
+    def add_condition_feature_pps_in_train_not_greater_than(self: SFC, threshold: float = 0.7) -> SFC:
```

Comment:
```suggestion
    def add_condition_feature_pps_in_train_not_greater_than(self: SFC, threshold: float = 0.2) -> SFC:
```
Or 0.3 dunno, but properties should not really be good at predicting the label.